boost-maven/boost-maven-plugin/src/it/test-run-server/src/test/java/it/LibertyRunMojoIT.java

```diff
@@ -22,30 +22,46 @@
 import java.io.InputStreamReader;
 
 public class LibertyRunMojoIT {
-    private static String URL;
+    private static String URL = "http://localhost:8080/";
 
     private static Process process;
 
     private static String runtimeGroupId;
     private static String runtimeArtifactId;
     private static String runtimeVersion;
 
+    private static String mvnCmd;
+
+    public static void setupMvnPath() {
+
+        String mvnPath = System.getenv("M2_HOME") + "/bin/mvn";
+
+        String osName = System.getProperty("os.name").toLowerCase();
+
+        if ((osName.indexOf("win") >= 0)) {
```

Comment:
Here, please use the OSUtil method to check for Windows: https://github.com/WASdev/ci.common/blob/master/src/main/java/net/wasdev/wlp/common/plugins/util/OSUtil.java